components/infobox/wikis/dota2/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,230 @@
+			Cell{name = 'Years Active (Talent)', content = {_args.years_active_talent}},
+		}
+	elseif id == 'history' then
+		if not String.isEmpty(_args.history_iwo) then
+			table.insert(widgets, Title{name = '[[Intel World Open|Intel World Open]] History'})
+			table.insert(widgets, Center{content = {_args.history_iwo}})
+		end
+		if not String.isEmpty(_args.history_gfinity) then
+			table.insert(widgets, Title{name = '[[Gfinity/Elite_Series|Gfinity Elite Series]] History'})
+			table.insert(widgets, Center{content = {_args.history_gfinity}})
+		end
+		if not String.isEmpty(_args.history_odl) then
+			table.insert(widgets, Title{name = '[[Oceania Draft League|Oceania Draft League]] History'})
+			table.insert(widgets, Center{content = {_args.history_odl}})
+		end
+	elseif id == 'role' then
+		return {
+			Cell{name = 'Current Role', content = {
+					CustomPlayer._createRole('role', _args.role),
+					CustomPlayer._createRole('role2', _args.role2)
+				}},
+		}
+	elseif id == 'nationality' then
+		return {
+			Cell{name = 'Location', content = {_args.location}},
+			Cell{name = 'Nationality', content = CustomPlayer._createLocations()}
+		}
+	end
+	return widgets
+end
+
+function CustomInjector:addCustomCells(widgets)
+	-- TODO add hero
+	return {}
+end
+
+function CustomPlayer:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomPlayer:makeAbbr(title, text)
+	if String.isEmpty(title) or String.isEmpty(text) then
+		return nil
+	end
+	return '<abbr title="' .. title .. '>' .. text .. '</abbr>'
+end
+
+function CustomPlayer:adjustLPDB(lpdbData)
+	lpdbData.status = lpdbData.status or 'Unknown'
+
+	lpdbData.extradata.role = _args.role
+	lpdbData.extradata.role2 = _args.role2
+	lpdbData.extradata.hero = _args.hero
+	lpdbData.extradata.hero2 = _args.hero2
+	lpdbData.extradata.hero3 = _args.hero3
+	lpdbData.extradata['lc_id'] = _base_page_name:lower()
+	lpdbData.extradata.team2 = mw.ext.TeamLiquidIntegration.resolve_redirect(
+		not String.isEmpty(_args.team2link) and _args.team2link or _args.team2)
+
+	return lpdbData
+end
+
+function CustomPlayer:createBottomContent(infobox)
+	if Namespace.isMain() then
+		return tostring(Matches.get({args = {noClass = true}}))
+	end
+end
+
+function CustomPlayer._getStatusContents()
+	local statusContents = {}
+	local status
+	if not String.isEmpty(_args.status) then
+		status = Page.makeInternalLink({onlyIfExists = true}, _args.status) or _args.status
+	end
+	table.insert(statusContents, status)
+
+	local banned = _BANNED[string.lower(_args.banned or '')]
+	if not banned and not String.isEmpty(_args.banned) then
+		banned = '[[Banned Players|Multiple Bans]]'
+		table.insert(statusContents, banned)
+	end
+
+	local index = 2
+	banned = _BANNED[string.lower(_args['banned' .. index] or '')]
+	while banned do
+		table.insert(statusContents, banned)
+		index = index + 1
+		banned = _BANNED[string.lower(_args['banned' .. index] or '')]
+	end
```

Comment:
```suggestion
	Table.mergeInto(statusContents, Player:getAllArgsForBase(_args, 'banned'))
```

Reply:
Oh wait, this doesn't do the lookup on _BANNED anymore, but that can be added

Reply:
Done